src/components/utilities/getResourceIdentifier.js

```diff
@@ -1,23 +1,7 @@
-import React from 'react';
-import { get } from 'lodash';
-
-import { Layout } from '@folio/stripes/components';
-
 export default function (eresource = {}, type) {
-  const identifiers = get(eresource, 'identifiers', []);
-  const entry = identifiers.find(i => i.identifier.ns.value === type);
-  if (!entry) return '';
-
-  const value = get(entry, 'identifier.value');
-  if (!value) return '';
-
-  if (Array.isArray(value)) {
-    return (
-      <Layout className="display-flex flex-direction-column">
-        {value.map((v, i) => <div key={i}>{v}</div>)}
-      </Layout>
-    );
-  }
+  const identifiers = eresource?.identifiers || [];
+  // find first identifiers value of namespace 'type'
+  const value = identifiers.find(i => i.identifier.ns.value === type)?.identifier?.value;
```

Comment:
This appears to no longer have [the array-support that was here](https://github.com/folio-org/ui-agreements/pull/541/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-0249252a51792ac88514adbaed7ad8b9L14-L20). Why is that no longer needed?

Reply:
Because `find` is returning one element, the first. So how could we ever have an array there?